sentry_sdk/consts.py

```diff
@@ -78,6 +78,7 @@ def __init__(
         auto_enabling_integrations=True,  # type: bool
         auto_session_tracking=True,  # type: bool
         send_client_reports=True,  # type: bool
+        enable_profiling=False,  # type: bool
```

Comment:
we decided this should move to experimental so please move it here
https://github.com/getsentry/sentry-python/blob/9e69fcf61a537ae30ea687f5b55957b6f3441935/sentry_sdk/consts.py#L29-L39
and change the checks in the context manager and tracing.py accordingly